### DISCUSSION TITLE
Local development using Docker

### DISCUSSION THREAD

**Discussion by gwatai** [brefphp/bref#1711] [Support]
Happy New Year, I have been having this error every time I try bref local development, The issue is am getting a max memory use 
![dockerComposerMaxMemoryUse](https://github.com/brefphp/bref/assets/90905286/1bd59ee3-65c3-4ca9-8302-dc02134837d3) which on the browser results to a "Failed to open stream: Permission denied" as shown in the screenshots below:
![DockerFilePermissionErrorOnBrowser](https://github.com/brefphp/bref/assets/90905286/dc46adb2-2712-4916-bc84-dc3f096d2c03)
when calling my API through Postman I get a file permissions error: 
![DockerFilePermissionError](https://github.com/brefphp/bref/assets/90905286/953f0373-1d76-47cb-96f0-fb9abf7ec779)
Below is my docker-compose.yml:
`version: "3.5"

services:
    app:
        image: bref/php-81-fpm-dev:2
        #host-container port mapping
        ports: [ '8000:8000' ]
        volumes:
            #host-container
                - .:/var/task:ro
                - ./storage:/var/task/storage
        environment:
            HANDLER: public/index.php
            # Assets will be served from this directory
            DOCUMENT_ROOT: public
`
dockerfile:
`#FROM --platform=arm64 bref/php-80-fpm:2
FROM bref/php-81-fpm:2

COPY . /var/task

#Configure the handler file (the entrypoint that receives all HTTP requests)
CMD ["public/index.php"]
`
Any assistance will be greatly appreciated 🥷

**Comment by LoreSco** [comment]
Hey @gwatai,
this fixed it for me https://bref.sh/docs/laravel/file-storage#public-files

**Reply by iljhli** [reply]
I'm running into the same issue, but the public files didn't do much since it can't seem to write to the /tmp/storage directory. It creates the folders, but then cannot write the files.

**Comment by iljhli** [comment]
I found this in an issue, and it worked for me:

https://github.com/brefphp/bref/issues/1644#issuecomment-1714783575
